var/spack/repos/builtin/packages/serf/package.py

```diff
@@ -19,13 +19,13 @@ class Serf(SConsPackage):
     variant('debug', default=False,
             description='Enable debugging info and strict compile warnings')
 
-    depends_on('scons@2.3.0:', type='build')
-
     depends_on('apr')
     depends_on('apr-util')
     depends_on('openssl')
-    depends_on('zlib')
+    depends_on('python')
```

Comment:
Should this be type build? Does it link to the Python libs/headers? Is it used at run-time?

Reply:
I do not know. The README does not even mention Python (or the available CMake build), there is no file called `INSTALL` or anything similar, and I am not familiar with SCons.

Reply:
The build log doesn't mention Python in any of the link lines and the resulting shared object library doesn't seem to link to Python, so I think we can make it `type='build'`.

Reply:
This sounds reasonable. I updated the build.